src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -3,9 +3,31 @@
 
```

Comment:
@cliffkoh can you provide a little more info here? Not quire sure what this comment is about and/or how to act on. Thanks.